### ISSUE TITLE
Bug: ckeditor toolbar disappears in super rare cases while editing

### ISSUE THREAD

**Issue by mhsdesign** [neos/neos-ui#3224]
### Description

Sorry, i hate to formulate such an issue without any real information.
But i talked to @NicoleNitschke at the sprint and she could confirm this issue.

So i will document this issue.

### Problem
While using the ckeditior the toolbar just dies in some rare cases -> and becomes black
after focusing another element and refocusing the previous ckeditor field - the toolbar is restored.

- im not sure anymore if i witnessed any console.errors but i think not.

Any further information would be great - if someone can reproduce this!!!

### Edit i could reproduce it:

UI version: v8.0.1

- you need to reload the Ui (to clear any js caches)
- then select the headline in the demo (directly)
- expect the toolbar to show
- click on the content collection - it resides in
- expect the toolbar to hide (as this is cust a content collection)
- click back to the headline
- expect the toolbar to show (again) -> FAILS!!!

[Bildschirmaufzeichnung vom 12.10.2022, 14:30:52.webm](https://user-images.githubusercontent.com/85400359/195343198-c342c04b-3d8a-4847-8cae-54d5af971127.webm)

### Affected Versions
UI:

i believe i saw it in:

... 7.1 - 8.0 ....

but its to expect this bug exists since all times

**Comment by grebaldi** [comment]
What browser does this happen in?

I tested FF, Chrome and WebKit (all on Linux) with Neos UI 7.3 and 8.0 and cannot reproduce. But since I'm seeing a different toolbar for that headline, specifics of the setup you've been using cannot be ruled out.

I have however seen a different problem that may or may not be related to this one:

[Peek 2023-01-05 14-00 - #3224.webm](https://user-images.githubusercontent.com/2522299/210786672-eb0f0c13-0ffa-4742-9f58-7bf3a6949e2d.webm)

The SelectBox header label of the formatting SelectBox in the rich text editor toolbar remains empty on occasion, even though a value should be present.

**Update:** I seem to be onto something here, because `formattingUnderCursor` appears to lag when switching between nodes (which explains the Issue I described just then, and may also explain the initial issue).

**Comment by mhsdesign** [comment]
thx will look into it ;)

**Comment by markusguenther** [comment]
Should be fixed with 7.3.12
